config/params.ini

```diff
@@ -1,6 +1,6 @@
 ; There is detailed documentation for each parameter in the README.md
 [general]
-name=Botty
```

Comment:
Don't change param defaults without really meaning to. Revert this file to original defaults and only add new fields